tests/integration/Social/Services/ReactionsCest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Kanvas\Packages\Tests\Integration\Social\Service;
+
+use Codeception\Lib\Di;
+use IntegrationTester;
+use Kanvas\Packages\Social\Services\Feeds;
+use Kanvas\Packages\Social\Services\MessageTypes;
+use Kanvas\Packages\Social\Services\Reactions;
+use Kanvas\Packages\Test\Support\Models\Users;
+
+class ReactionsCest
+{
+    public $reaction;
+
+    /**
+     * Create a reaction for testing
+     *
+     * @return void
+     */
+    protected function reactionCreation(): void
+    {
+        $this->reaction = Reactions::createReaction('test-reaction', new Users, "💀");
```

Comment:
@arfenis on the test I normally pass a user:: findfirst () instead of just a new instance